src/Event/Productions/ProductionRepository.php

```diff
@@ -57,7 +57,7 @@ public function addEvent(string $eventId, Production $production)
                 'event_id' => $eventId,
                 'production_id' => $production->getProductionId()->toNative(),
                 'name' => $production->getName(),
-                'added_at' => $addedAt->format(DATE_ATOM),
+                'added_at' => $addedAt->format('Y-m-d'),
```

Comment:
Why were these changed? We're losing some info now (exact time).

Reply:
Because it would break. DB would throw an error

Reply:
How come this didn't fail in the unit tests in that case?

Reply:
Could be because test use sqlite.
This is what I get (when pinging the endpoint so it's trying to persist to MySQL)
```
{
    "title": "An exception occurred while executing 'INSERT INTO productions (event_id, production_id, name, added_at) VALUES (?, ?, ?, ?)' with params [\"44e35903-0dc0-4cfd-8a1e-764a9f5436ba\", \"d8fcb590-79c1-49ed-afb8-81b0909eb443\", \"Marsonia\", \"2020-07-06T11:04:57+02:00\"]:\n\nSQLSTATE[22007]: Invalid datetime format: 1292 Incorrect date value: '2020-07-06T11:04:57+02:00' for column 'added_at' at row 1",
    "type": "about:blank",
    "status": 400
}
```

Reply:
Makes sense. We're using the `date_immutable` type from Doctrine DBAL here. I don't think we ever wanted to have more granularity than a single day. The field is mainly here to e.g. only search for "recent" productions.